src/duplicacy_gcdstorage.go

```diff
@@ -45,11 +46,19 @@ type GCDConfig struct {
 }
 
 func (storage *GCDStorage) shouldRetry(threadIndex int, err error) (bool, error) {
+	const LIMIT_BACKOFF_TIME = 64
+	const MAX_NUMBER_OF_RETRIES = 15
+	minimumSleepRatio := 0.1
+	maximumSleepRatio := 0.2
+	minimumSleep := float64(storage.numberOfThreads) * minimumSleepRatio
```

Comment:
How about a comment explaining why you're using this algorithm (threads * 0.1 or 0.2) and what this particular algorithm is trying to accomplish. As-is, a casual reviewer has no way to know why we're doing this nor whether it's the right thing to be doing.